### DISCUSSION TITLE
Unable to add multiple roles per level

### DISCUSSION THREAD

**Discussion by ThunderMite42** [danthe1st/Mee6-bypasser#66] [Support] [answered]
My old MEE6 setup would give two roles at level 10, but whenever I try adding the second role it automatically removes the first (and I know it's not the role removal toggle, because I have it set so that roles stack). Is there a way to set one level to give multiple roles?

**Comment by noobidev** [comment, marked as answer]
In the current state it is not possible to add multiple roles upon reaching a specific level. The only way is to add them to prior levels.
